assets/scss/pages/contact-page.scss

```diff
@@ -0,0 +1,109 @@
+.c-contact-page__section {
```

Comment:
@mgbelegu same as above. Please the `p-` prefix. The class name should be something like: `p-contact`, `p-contact__section`, etc.